### DISCUSSION TITLE
[Feature] Give better control to disable panorama view for certain photos

### DISCUSSION THREAD

**Discussion by amykyta3** [immich-app/immich#14582] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

 It would be nice if for these partial panoramas, there was a way to disable the panorama view. Many of my "panoramas" are really just stitches of 2-3 images to slightly expand the FOV and display much better in a "flat" view.

Example below, (after using the zoom in/out workaround to getting past the bug described in #10917)
![image](https://github.com/user-attachments/assets/693fa31e-c63a-4604-9652-a5aa9f4d15f8)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3854
